app/views/editor/questionRich.html.erb

```diff
@@ -0,0 +1,313 @@
+
+        </div>
+      </div>
+
+    </div>
+    <!-- end main_image module -->
+
+
+    <!-- body module -->
+    <div class="ple-module-body ple-module container">
+
+      <div class="ple-module-guide col-md-3">
+        <h2>3</h2>
+      </div>
+
+      <div class="ple-module-content col-md-9 container">
+        <textarea id="text-input" class="ple-textarea form-control"><% if @node && @node.latest && @node.latest.body %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
+      </div>
+
+    </div>
+    <!-- end body module -->
+
+
+    <!-- tags module -->
+    <div class="ple-module-tags ple-module container">
+
+      <div class="ple-module-guide col-md-3">
+        <h2>4</h2>
+      </div>
+
+      <div class="ple-module-content col-md-9">
+        <input class="form-control input-lg" type="text" placeholder="balloon-mapping, water-quality..." value="<% if params[:tags] || (@node && @node.tagnames) %><%= params[:tags] || @node.tagnames.join(',') %><% end %>" />
+      </div>
+
+    </div>
+    <!-- end tags module -->
+
+    <!-- multi module -->
+    <!--
+    <div class="ple-module-multi ple-module container">
+
+      <div class="ple-module-guide col-md-3">
+        <p>Extras</p>
+      </div>
+
+      <div class="ple-module-content col-md-9">
+
+        <div class="btn-group">
+          <a class="btn btn-lg btn-default">
+            <i class="fa fa-comment-o"></i> Invite responses <i class="fa fa-caret-down"></i>
+          </a>
+          <a class="btn btn-lg btn-default">
+            <i class="fa fa-envelope-o"></i> Event <i class="fa fa-caret-down"></i>
+          </a>
+          <a class="btn btn-lg btn-default">
+            <i class="fa fa-map-o"></i> Map
+          </a>
+          <a class="btn btn-lg btn-default">
+            <i class="fa fa-line-chart"></i> Chart
+          </a>
+          <a class="btn btn-lg btn-default">
+            More <i class="fa fa-caret-down"></i>
+          </a>
+        </div>
+
+      </div>
+
+    </div>
+    -->
+    <!-- end multi module -->
+  
+
+  </div>
+
+  <div class="ple-footer">
+
+    <div class="ple-menu-more" style="display:none;">
+      <b>History</b>
+      <div class="ple-history">
+      </div>
+      <hr />
+      <!--
+      <div class="form-control input-md ple-history">
+        <option selected value="">recent edits</option>
+      </div>
+      -->
+    </div>
+
+    <button class="btn btn-lg btn-default btn-more">...</button>
+
+    <span> &nbsp; By publishing, you agree to <a href="https://publiclab.org/licenses">open source your work</a><span class="hidden-xs"> so that others may use it.</span></span>
+
+    <button class="ple-publish btn btn-lg btn-primary pull-right disabled">Publish</button>
+    <span class="ple-help pull-right">
+      <span class="ple-steps-left">2</span>
+      <span class="hidden-xs"> steps left</span>
+    </span>
+    
+  </div>
+
+</div>
+  <%= javascript_include_tag "post" %>
```

Comment:
See how here, the new "rich" template ends and the older editor template begins? You actually only need one of the two -- that's why you're seeing the old form sort of overlaid on the new one. You should be able to remove everything below this line, but transfer some of the special question-related features from the old template to the new. I'll mark the ones I see.